pkg/git/gitprovider/common.go

```diff
@@ -33,3 +39,11 @@ func IsInsecureSSL() bool {
 	}
 	return false
 }
+
+// CheckGitUrlError returns more specific git error
+func CheckGitUrlError(err error) error {
```

Comment:
This must be implemented in each git provider because:
1. We adding dependency for a specific internal package
2. Different providers might have different responses

Reply:
moved